packages/manager/modules/pci/src/projects/project/notebooks/notebook.constants.js

```diff
@@ -47,6 +47,12 @@ export const NOTEBOOK_STORAGE_INFO = {
 
 export const NOTEBOOK_MULTIPLY_SIGN = ' × ';
 
+export const NOTEBOOK_FREE_WORKSPACE_STORAGE_DEFAULT = 10737418420;
+
+export const NOTEBOOK_MINUTES_IN_HOUR = 60;
```

Comment:
NOTEBOOK_MINUTES_IN_HOUR = 60
NOTEBOOK_MULTIPLY_SIGN = ' × '

i think we have gone a little bit too far with constants on this one :)